src/components/Markdown.svelte

```diff
@@ -13,6 +13,15 @@
         start ? `start="${start}"` : ""
       } class="mzp-u-list-styled">${body}</${outerEl}>`;
     },
+    heading(text, level, raw) {
+      const id = raw
+        .toLowerCase()
+        .trim()
+        .replace(/<[!\/a-z].*?>/gi, '')
+        .replace(/ +/g, '-');
+
+      return `<h${level} class="annotation-header-link" id="${id}"><a href="#${id}">${text}</a></h${level}>\n`;
```

Comment:
> Probably should make the headings black and underlined on hover.

what do you think if we add inline styling here:

```suggestion
      return `<h${level} class="annotation-header-link" id="${id}"><a href="#${id}" style="color: black; text-decoration: none;" onmouseover="this.style.textDecoration='underline'" onmouseout="this.style.textDecoration='none'">${text}</a></h${level}>\n`;
```

Reply:
yeah it's not the prettiest, but unfortunately since this is a dynamically generated component, any content generated by this function and subsequently rendered in the markup (HTML) will not automatically inherit CSS styles, unless they are explicitly scoped, like inline styling as i suggested here.

Reply:
yeah ... I was not happy with that. We should need inline JavaScript for styling. So I added styling now.